packages/enzyme-test-suite/test/ReactWrapper-spec.jsx

```diff
@@ -5891,14 +5891,18 @@ describeWithDOM('mount', () => {
           }
 
           render() {
-            const { throws } = this.state;
+            const {
+              didThrow,
+              throws,
+            } = this.state;
+
             return (
               <div>
                 <MaybeFragment>
                   <span>
                     <Thrower throws={throws} />
                     <div>
-                      {this.state.didThrow ? 'HasThrown' : 'HasNotThrown'}
+                      { didThrow ? 'HasThrown' : 'HasNotThrown' }
```

Comment:
jsx curlies don't get bounding spaces
```suggestion
                      {didThrow ? 'HasThrown' : 'HasNotThrown'}
```